client/a8c-for-agencies/components/onboarding-tour-modal/index.tsx

```diff
@@ -54,10 +59,35 @@ function OnboardingTourModal( { onClose, children }: OnboardingTourModalProps )
 	);
 
 	const actions = useMemo( () => {
-		return currentSection?.props?.renderActions?.( {
-			onClose,
-			onNext: () => setCurrentSectionId( sections[ currentSectionIndex + 1 ]?.props?.id ),
-		} );
+		return currentSection?.props
+			?.renderableActions?.( {
+				onClose,
+				onNext: () => setCurrentSectionId( sections[ currentSectionIndex + 1 ]?.props?.id ),
+			} )
+			?.map( ( renderableAction: RenderableAction ) => {
+				if ( ! renderableAction ) {
+					return null;
+				}
+
+				if ( isValidElement( renderableAction ) ) {
+					return renderableAction;
+				}
+
+				const buttonProps = renderableAction as ActionProps;
+
+				return (
+					<Button
+						key={ buttonProps.label }
+						href={ buttonProps.href }
+						onClick={ buttonProps.onClick }
+						variant={ buttonProps.variant }
+						disabled={ buttonProps.disabled }
+						isBusy={ buttonProps.isBusy }
+					>
+						{ buttonProps.label }
+					</Button>
+				);
```

Comment:
We could probably simplify this. 

```suggestion
				const { label, ...restButtonProps } = buttonProps;

				return (
					<Button
						key={ label }
						{ ...restButtonProps }
					>
						{ label }
					</Button>
				);
```

Reply:
Good suggestion. This has been applied 👍